### ISSUE TITLE
I hear no audio during video calls with a contract

### ISSUE THREAD

**Issue by t3io** [WhisperSystems/Signal-Android#6271]
I have:
- [ x] searched open and closed issues for duplicates
- [x ] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
In video calls with a specific contact I receive no audio while my friend can hear me. 

### Steps to reproduce
- start a beta video-call
- video appears fine
- the other side can hear me
- I can't hear them

**Actual result:** I can't hear them
**Expected result:** I hear them. This is what happens with other contacts. 

### Device info
**Device model:** Samsung Galaxy S5 Neo 
**Android version:** 6.0.1
**Signal version:** 3.29.6

### Link to debug log
After first attempt to chat with the contact:
https://gist.github.com/d0cdff23b380a3899762ccf15712365d

After second attempt:https://gist.github.com/b5a8ca7409b8fd717f01e20a6292ae8a

**Comment by 2-4601** [comment]
Could be related to #6140.

**Comment by AlfonsoMuskedunder** [comment]
Do you happen to know if third parties can hear your contact when he/she calls them?

**Comment by t3io** [comment]
It appears to be an intermittent issue. Just called from another phone & Signal account and I could video-call my friend perfectly. I then tried once more (the 3rd time now) with my own phone (which generated the attached log) and all was fine. Go figure. Potential networking issue? Anything useful in this issue, or should we just dismiss this as a (frequent) fluke?

**Comment by Zentriert** [comment]
I have an LG G5 with Android 7.0 and my friend has an iPhone 7+ as well as an unknown Android device and we've experienced this issue on and off too. 

Ending the call and making a new one or restarting our devices seems to fix the issue, but it can be annoying.

**Comment by moxie0** [comment]
I think this should work more consistently in 3.31.0 now. Please reopen if problems persist w/ that branch

**Comment by SomeStrangeName** [comment]
I got such an issue today with two Sony Xperia Z smartphones.
A) with Android 5.1.1 from Sony and 12MBit/s internet connection
B) with LineageOS 14.2 and slower internet connection

Video was working in both directions video from B) was not very sharp (I assume because of slow connection).
Voice was working from A) -> B) but not from B) -> A).

Call was started from A) -> B).

Tested it two times.

Normal phone call was working (so microphone is not defect :-)).

Each phone is using latest Signal version.

Maybe this helps?
